django/contrib/admin/static/admin/js/SelectFilter2.js

```diff
@@ -6,6 +6,7 @@ Requires core.js and SelectBox.js.
 */
 'use strict';
 {
+    const $ = django.jQuery;
```

Comment:
We try to remove `jQuery` usage, see 3a807a6f597a97b0fa514039a8ea41fb90769981. Please don't introduce it again in the `SelectFilter2.js`.